modules/mono/editor/bindings_generator.cpp

```diff
@@ -1282,6 +1290,8 @@ Error BindingsGenerator::_generate_cs_type(const TypeInterface &itype, const Str
 				  "#pragma warning disable CS1573 // Disable warning: "
 				  "'Parameter has no matching param tag in the XML comment'\n");
 
+	output.append("\n#nullable disable\n");
```

Comment:
I think it's better this way. Not really a problem adding the `#nullable disable` line, and we never know, maybe in the future the Godot scripting API could provide nullable information :crossed_fingers: (I wouldn't count on that though)

Reply:
I think this is highly unlikely to happen since the concept of nullable types does not exist in GDScript ([yet?](https://github.com/godotengine/godot-proposals/issues/162)).

Not sure if the compiler treats generated code differently in other aspects other than the nullability being disabled, which is why I brought it up, but if it's the same I don't mind using `#nullable disable`